watson_developer_cloud/text_to_speech_v1.py

```diff
@@ -12,123 +14,1162 @@
+        args = {}
+        if 'customization_id' in _dict:
+            args['customization_id'] = _dict['customization_id']
+        else:
+            raise ValueError(
+                'Required property \'customization_id\' not present in VoiceModel JSON'
+            )
+        if 'name' in _dict:
+            args['name'] = _dict['name']
+        if 'language' in _dict:
+            args['language'] = _dict['language']
+        if 'owner' in _dict:
+            args['owner'] = _dict['owner']
+        if 'created' in _dict:
+            args['created'] = _dict['created']
+        if 'last_modified' in _dict:
+            args['last_modified'] = _dict['last_modified']
+        if 'description' in _dict:
+            args['description'] = _dict['description']
+        if 'words' in _dict:
+            args['words'] = [Word._from_dict(x) for x in _dict['words']]
+        return cls(**args)
+
+    def _to_dict(self):
+        """Return a json dictionary representing this model."""
+        _dict = {}
+        if hasattr(self,
+                   'customization_id') and self.customization_id is not None:
+            _dict['customization_id'] = self.customization_id
+        if hasattr(self, 'name') and self.name is not None:
+            _dict['name'] = self.name
+        if hasattr(self, 'language') and self.language is not None:
+            _dict['language'] = self.language
+        if hasattr(self, 'owner') and self.owner is not None:
+            _dict['owner'] = self.owner
+        if hasattr(self, 'created') and self.created is not None:
+            _dict['created'] = self.created
+        if hasattr(self, 'last_modified') and self.last_modified is not None:
+            _dict['last_modified'] = self.last_modified
+        if hasattr(self, 'description') and self.description is not None:
+            _dict['description'] = self.description
+        if hasattr(self, 'words') and self.words is not None:
+            _dict['words'] = [x._to_dict() for x in self.words]
+        return _dict
+
+    def __str__(self):
+        """Return a `str` version of this VoiceModel object."""
+        return json.dumps(self._to_dict(), indent=2)
+
+    def __eq__(self, other):
+        """Return `true` when self and other are equal, false otherwise."""
+        if not isinstance(other, self.__class__):
+            return False
+        return self.__dict__ == other.__dict__
+
+    def __ne__(self, other):
+        """Return `true` when self and other are not equal, false otherwise."""
+        return not self == other
+
+
+class VoiceModels(object):
+    """
+    VoiceModels.
+
+    :attr list[VoiceModel] customizations: An array of `VoiceModel` objects that provides information about each available custom voice model. The array is empty if the requesting service credentials own no custom voice models (if no language is specified) or own no custom voice models for the specified language.
```

Comment:
Seems like we want this to be named `voiceModels` and not `customizations`.  I think this would be as simple as adding an `x-alternate-name` annotation to the swagger.